migrate-the-clay-components-from-v2-to-v3.md

```diff
@@ -0,0 +1,23 @@
+# Migrating clay components from v2 to v3
+
+Noteable Changes
+
+## clay-button
+
+-   Removed icon dependency within button itself
+
+```js
```

Comment:
Maybe using `diff` would have made it clearer, what do you think?

```diff
<Button
-	label="Icon positioned at right"
-	icon="plus"
-	iconAlignment="right"
-	spritemap={spritemap}
-/>
+>
+      {'Icon positioned at right'}
+      <Icon spritemap={spritemap} symbol="plus"/>
+</Button>
```

Reply:
Personally I would probably avoid using diffs in documentation just because I think they can be heard to read. But if you think it'd be a big help, I'm happy to do whatever.

Reply:
I think it mostly comes down to size: in small examples like this, I can easily see what has changed from v2 to v3 and seeing the examples written out in full is nice and easy. On the other hand, if the examples are large, then it will be hard for me to see exactly what's different without the aid of a diff. Hopefully we won't have too many that are like that, but we can evaluate those when we come to them.

Reply:
@bryceosterhaus I do not see problem in not using in some cases like this, but for me it becomes clearer when I see a diff, some not so much ... but I agree with @wincent on bigger changes (I also hope that we do not change many things) maybe is necessary.